source/assets/stylesheets/ie8_fixes.scss

```diff
@@ -31,12 +33,23 @@
 }
 
 // Place icons in the middle of its container
-.footer-social_icons__icon {
+.social_icons__icon {
   padding-top: 6px;
   height: 22px;
 }
 
 // To support full coloured div
-.footer-social-icons {
+.social-icons {
   display: block;
 }
+
+// To set a blog article width
+.blog-header__article-header,
+.blog-article__container,
+.blog-author__container,
+.blog-comments__container,
+.blog-related__container,
+.blog-related__header {
```

Comment:
Also I got hands a little bit tight re containers as the body colour is set to `#F7F7F7` and blog container is `white` so whenever I'll set width to the main blog container it will show background body colour instead of white. That's why there is a bit fuss with containers on blog post page.